src/index.js

```diff
@@ -8,6 +8,7 @@ import RedBox from './components/RedBox';
 import View from './components/View';
 import Text from './components/Text';
 import TextStyles from './sharedStyles/TextStyles';
+import Symbol from './symbol';
```

Comment:
I renamed to `makeSymbol` and `injectSymbols`, both imported from the top namespace. The example in the PR description is updated to demonstrate. Happy to change these as many times as needed until it feels right.